File: model/Candidato.js
class Candidato {
    constructor(nome, email, password, cpf, celular, cep, logradouro, bairro, cidade, estado, pais, area){
        this._id;
        this._nome = nome;        
        this._email = email;
        this._password = password;
        this._cpf = cpf;
        this._celular = celular;
        this._cep = cep;
        this._logradouro = logradouro;
        this._bairro = bairro;
        this._cidade = cidade;
        this._estado = estado;
        this._pais = pais;
        this._area = area;
    }

    get id(){
        return this._id;
    }
    
    get nome(){
        return this._nome;
    }

    get email(){
        return this._email;
    }

    get password(){
        return this._password;
    }

    get cpf(){
        return this._cpf;        
    }

    get celular(){
        return this._celular;
    }

    get cep(){
        return this._cep;
    }
    
    get logradouro(){
        return this._logradouro;
    }

    get bairro(){
        return this._bairro;
    }
     
    get cidade(){
        return this._cidade;
    }
    
    get estado(){
        return this._estado;
    }
    
    get pais(){
        return this._pais;
    }
    
    get area(){
        return this._area;
    }

}

File: backend/src/main/java/com/g5tech/api/model/Candidato.java
package com.g5tech.api.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.util.Date;

/**
 * Candidato --- representa um candidato no sistema.
 */

@Entity
@Table(name = "tb_candidato")
@Getter
@Setter
public class Candidato {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    @Column(name = "nome")
    private String nome;

    @Column(name = "cpf")
    private String cpf;

    @Column(name = "celular")
    private String celular;

    @Column(name = "cep")
    private String cep;

    @Column(name = "rua")
    private String rua;

    @Column(name = "bairro")
    private String bairro;

    @Column(name = "cidade")
    private String cidade;

    @Column(name = "estado")
    private String estado;

    @Column(name = "pais")
    private String pais;

    @Column(name = "area")
    private String area;

    @Column(name = "genero")
    private String genero;

    @Column(name = "data_nascimento")
    private Date dataNascimento;

    @Column(name = "email")
    private String email;
}
